dcpy/utils/s3.py

```diff
@@ -231,13 +231,13 @@ def download_folder(
     prefix: str,
     export_path: Path,
     *,
-    include_prefix_in_export: bool = False,
+    include_prefix_in_export: bool = True,
```

Comment:
Swapping the default, so that nothing changes for existing callers. (No previous callers specified this kwarg)